tracing/decorators/entrypoint.py

```diff
@@ -181,35 +178,38 @@ def _generate_and_serialize_performance_data(
     mocks: dict,
     substituted_output: str,
 ) -> pd.DataFrame:
-    tracers = getattr(registered_call, constants.TRACERS_ATTRIBUTE)
+    tracers: list[Tracer] = getattr(registered_call, constants.TRACERS_ATTRIBUTE)
     measured_times = np.zeros(
         (1 + len(tracers), constants.AMOUNT_EXECUTIONS_TESTING_PERFORMANCE)
     )
-    proj_path = tracers[0].proj_path
     for i in range(constants.AMOUNT_EXECUTIONS_TESTING_PERFORMANCE):
+        # Normal test execution, no tracing
+        start_time = default_timer()
         if clazz is None:
-            start_time = default_timer()
             registered_call(**mocks)
         else:
             instance: typing.Any = object.__new__(clazz)
-            start_time = default_timer()
             registered_call(instance, **mocks)
```

Comment:
I recommend placing the start time as close to the callable. We don't want to measure the time of the if check and object instantiation. The additional noise is undesirable for performance evaluation, especially for callable execution without tracing, where the time elapsed can be within 1e-7 - 1e-6.